nexus/src/app/instance_network.rs

```diff
@@ -438,6 +438,7 @@ impl super::Nexus {
     pub(crate) async fn instance_delete_dpd_config(
         &self,
         opctx: &OpContext,
+        opctx_alloc: &OpContext,
```

Comment:
I think the usage of this `opctx_alloc` is correct - as in, it appears to be the right opctx we should use -- but this is a method of the `Nexus ` struct which contains the `opctx_alloc` field already (see: `nexus/src/app/mod.rs` for the definition).

Could we just access `self.opctx_alloc`? Seems weird to have it be passed as a parameter, if we don't really want the caller to customize it.

Reply:
Oh! Yes, that would make sense.